spec/acceptance/class_spec.rb

```diff
@@ -97,12 +105,36 @@ class { 'agent_upgrade': }
       it { is_expected.to be_enabled }
       it { is_expected.to be_running }
     end
+
+    it 'should have mcollective correctly configured' do
+        on default, '/opt/puppetlabs/bin/mco ping' do
+          assert_match(/^#{default}\s+time=/, stdout)
+        end
+    end
+
+    describe file('/etc/puppetlabs/mcollective/server.cfg') do
+      it { is_expected.to exist }
+      its(:content) {
+        is_expected.to include 'libdir = /usr/libexec/mcollective/plugins:/opt/puppetlabs/mcollective/plugins'
+        is_expected.to include 'logfile = /var/log/puppetlabs/mcollective.log'
+        is_expected.to include 'plugin.yaml = /etc/mcollective/facts.yaml:/etc/puppetlabs/mcollective/facts.yaml'
```

Comment:
I see the ticket mentions to "Append the new libdir location to libdir" and we do that, but I'm wondering if that's the right thing to do?

Seems like the current behavior could be bad if you had an older version of an mco plugin in the old place (perhaps not compatible with puppet 4), and a newer version of the plugin in the new place, and we'd give precedence to the older one.

But perhaps mco doesn't allow multiple versions of the same plugin in the `libdir` and so this would never be a concern? /cc @richardc